scylla/src/transport/iterator.rs

```diff
@@ -248,11 +248,12 @@ impl RowIterator {
                 }
             };
 
+            let kstable = config.prepared.get_table_spec();
             let statement_info = RoutingInfo {
                 consistency,
                 serial_consistency,
                 token,
-                keyspace: config.prepared.get_keyspace_name(),
+                table: kstable,
```

Comment:
OK, then choose the convention you find the most reasonable and stick to it.

Reply:
Renamed `TableSpec` arguments to `table_spec` - `table` / `keyspace` are used for strings. One exception: `RoutingInfo` field is still called `table`, I think that changing it would harm readability so I decided against it.